be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -302,6 +304,20 @@ void LakeTabletsChannel::add_chunk(Chunk* chunk, const PTabletWriterAddChunkRequ
         int64_t tablet_id = tablet_ids[row_indexes[from]];
         auto& dw = _delta_writers[tablet_id];
         DCHECK(dw != nullptr);
+
+        // back pressure OlapTableSink since there are too many memtables need to flush
+        while (dw->queueing_memtable_num() >= config::max_queueing_memtable_per_tablet) {
+            auto t1 = std::chrono::steady_clock::now();
+            if (std::chrono::duration_cast<std::chrono::microseconds>(t1 - t0).count() / 1000 > request.timeout_ms()) {
+                LOG(INFO) << "LakeTabletsChannel txn_id: " << _txn_id << " load_id: " << print_id(request.id())
+                          << " wait tablet " << tablet_id << " flush memtable " << request.timeout_ms()
+                          << "ms still has queueing num " << dw->queueing_memtable_num();
+                break;
+            }
+            bthread_usleep(10000); // 10ms
+            wait_memtable_flush_time_us += 10000;
```

Comment:
bthread_usleep may sleep quite longer than expected when the bthread workers are busy

Reply:
any suggestion about this?
In addition, if the bthread worker is really full, then it does not seem to be a big problem if the rpc request is blocked for a longer time.

Reply:
How about `wait_memtable_flush_time_us += now() - t1`

Reply:
done, misunderstanding before.